website/src/app/dashboard/create-dialog.html

```diff
@@ -1,4 +1,4 @@
-<h2 mat-dialog-title>Create incident for {{data?.issue?.room}}</h2>
+<h2 mat-dialog-title>Create incident for {{data?.issue?.room?.id}}</h2> <!-- Fixed: printing [object, Object]-->
```

Comment:
Let's remove the comment here.
```suggestion
<h2 mat-dialog-title>Create incident for {{data?.issue?.room?.id}}</h2>
```